ipn/ipnlocal/local.go

```diff
@@ -4649,24 +4649,27 @@ func (b *LocalBackend) sshServerOrInit() (_ SSHServer, err error) {
 
 var warnSSHSELinux = health.NewWarnable()
 
-func checkSELinux() {
+// CheckSELinux reports whether SELinux is in "Enforcing" mode.
+// If it is, it warns the user that Tailscale SSH may not work.
+func CheckSELinux() bool {
```

Comment:
weird func. It both reports a value and has some side effects. Document more?

Reply:
Thus a draft

Reply:
split in two